static/app/components/modals/widgetViewerModal/widgetViewerTableCell.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {Location, LocationDescriptorObject} from 'history';
+
+import SortLink from 'sentry/components/gridEditable/sortLink';
+import Tooltip from 'sentry/components/tooltip';
+import Truncate from 'sentry/components/truncate';
+import {Organization, PageFilters} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {getIssueFieldRenderer} from 'sentry/utils/dashboards/issueFieldRenderers';
+import {TableDataRow, TableDataWithTitle} from 'sentry/utils/discover/discoverQuery';
+import {isFieldSortable} from 'sentry/utils/discover/eventView';
+import {getFieldRenderer} from 'sentry/utils/discover/fieldRenderers';
+import {
+  fieldAlignment,
+  getAggregateAlias,
+  getEquationAliasIndex,
+  isEquationAlias,
+} from 'sentry/utils/discover/fields';
+import {DisplayType, Widget, WidgetType} from 'sentry/views/dashboardsV2/types';
+import {eventViewFromWidget} from 'sentry/views/dashboardsV2/utils';
+import {ISSUE_FIELDS} from 'sentry/views/dashboardsV2/widgetBuilder/issueWidget/fields';
+import TopResultsIndicator from 'sentry/views/eventsV2/table/topResultsIndicator';
+import {TableColumn} from 'sentry/views/eventsV2/table/types';
+
+// Dashboards only supports top 5 for now
+const DEFAULT_NUM_TOP_EVENTS = 5;
+
+type Props = {
+  location: Location;
+  organization: Organization;
+  selection: PageFilters;
+  widget: Widget;
+  tableData?: TableDataWithTitle;
+};
+
+export const renderGridHeaderCell =
+  ({selection, widget, tableData}: Props) =>
+  (column: TableColumn<keyof TableDataRow>, _columnIndex: number): React.ReactNode => {
```

Comment:
```suggestion
  (column: GridColumnOrder<ObjectKey>, _columnIndex: number): React.ReactNode => {
```

🤷‍♀️

Reply:
I wasn't able to reduce the type of `renderGridHeaderCell` 😞 . `renderGridHeaderCell` requires a `type` attribute on the `column` arg which is not available in `GridColumnOrder`.

I've updated the type assertion for `renderGridHeaderCell` in `widgetViewerModal` to be more precise for now so we aren't using `any`.

I think we'll probably need to make some changes to `gridEditable/index` if we want to clean this up, which might be something for a different pr. Let me know what you think!

Reply:
I understand. Yes, we can address it in a follow-up PR....thanks for looking into that